src/stan_inference.jl

```diff
@@ -62,6 +62,7 @@ function stan_inference(prob::DiffEqBase.DEProblem,
                         # stan_sample keyword arguments
                         num_samples = 1000, num_warmups = 1000,
                         num_cpp_chains = 1, num_chains = 1, num_threads = 1,
+                        delta=0.8,
```

Comment:
Is that the default value if it is not specified?

Reply:
To make the formatter happy (I guess):
```suggestion
                        delta = 0.8,
```